xdn-elasticpath-example/deploying-Elastic-Path-on-XDN.md

```diff
@@ -0,0 +1,243 @@
+# Deploying Elastic Path on XDN
+
+## 1. Install Elastic Path
+
+- Clone https://github.com/elasticpath/react-pwa-reference-storefront
+- Navigate to the project folder: `cd %project_name%`
+- Install packages: `yarn`
+- In `src/ep.config.json` file: replace a host of `skuImagesUrl` and `siteImagesUrl` to whatever host you'll be using, for example:
+```json
+"skuImagesUrl": "https://moovweb-demos-elasticpath-default.moovweb-edge.io/BELLEVIE/skuImages/%fileName%",
+"siteImagesUrl": "https://moovweb-demos-elasticpath-default.moovweb-edge.io/BELLEVIE/siteImages/%fileName%",
+```
+- Build Elastic Path app: `yarn build` (`build` folder is going to be created)
+
+## 2. Install XDN
+
+- Install XDN packages: `yarn add -D @xdn/cli@latest && yarn add @xdn/core@latest @xdn/prefetch@latest @xdn/devtools@latest`
+- Next steps will cover the XDN key files you need to add. For more information about them see https://developer.moovweb.com/guides/starter
+- Add `xdn.config.js` file with your origin and images hosts. For example:
+```js
+module.exports = {
+  routes: './xdn/routes.js',
+  backends: {
+    origin: {
+      domainOrIp: 'reference81.epdemos.com',
+      hostHeader: 'reference81.epdemos.com',
+    },
+    images: {
+      domainOrIp: 'ep-demo-assets.s3-us-west-2.amazonaws.com',
+      hostHeader: 'ep-demo-assets.s3-us-west-2.amazonaws.com',
+    },
+  },
+};
+```
+
+- Add `xdn/service-worker.js` file:
+```js
```

Comment:
@egormesyats I think we should do the same here, show the power of `xdn init` to install these packages and creating the files for us.